lnbits/wallets/corelightning.py

```diff
@@ -174,7 +174,9 @@ async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse
                 "description": invoice.description,
             }
 
-            r = await run_sync(lambda: self.ln.call(self.pay, payload))
+            # Use configured pay command (e.g. xpay), but default to "pay"
+            pay_command = self.pay or "pay"
```

Comment:
```    corelightning_pay_command: str = Field(default="pay")```
self.pay should never be none, maybe check some like` if self.pay not in ["pay", "xpay", "mpay"]: raise` or create a `self.valid_pay_commands` or even better let the config by a select box instead of a string